extensions/PhabBugz/lib/Util.pm

```diff
@@ -64,6 +66,24 @@ sub get_revisions_by_ids {
     return @{$result->{result}{data}};
 }
 
+sub get_revisions_by_phids {
+    my ($phids) = @_;
+
+    my $data = {
+        queryKey => 'all',
+        constraints => {
+            phids => $phids
+        }
+    };
+
+    my $result = request('differential.revision.search', $data);
+
+    ThrowUserError('invalid_phabricator_revision_id')
+        unless (exists $result->{result}{data} && @{ $result->{result}{data} });
+
+    return @{$result->{result}{data}};
```

Comment:
if this is potentially large, it might be best to return the reference and reference it at the place it is used. Otherwise it will do a copy of the full array.